core/client/fs/src/test/java/alluxio/client/lineage/AlluxioLineageTest.java

```diff
@@ -25,7 +25,7 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.mockito.Mockito;
+import static org.mockito.Mockito;
```

Comment:
put `import static org.mockito.Mockito` before all other non-static import in a separate section.

Reply:
Thank you for your advice. 
 I have corrected that file and re-PRed.

Reply:
I have rewritten the file according to what you said ,but it is still wrong.
Could you tell me what's wrong exactly with it?
Thanks.